src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -366,13 +366,16 @@ protected void buildCraftingRecipes(Consumer<FinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(MSItems.EIGHTBALL.get()).grist(TAR, 888).grist(COBALT, 888).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSItems.DICE.get()).grist(BUILD, 26).grist(CHALK, 6).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSBlocks.MIRROR.get()).grist(BUILD, 20).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.GUTTER_BALL.get()).grist(BUILD, 11262014).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.GUTTER_THUMB_DRIVE.get()).grist(BUILD, 2000000).grist(AMETHYST, 2000000).grist(MARBLE, 2000000).grist(RUBY, 2000000).grist(COBALT, 2000000).grist(DIAMOND, 2000000).build(recipeSaver);
 		
 		GristCostRecipeBuilder.of(MSItems.SBURB_CODE.get()).grist(CHALK, 16).grist(IODINE, 2).buildFor(recipeSaver, Minestuck.MOD_ID);
 		GristCostRecipeBuilder.of(MSItems.COMPLETED_SBURB_CODE.get()).grist(CHALK, 16).grist(IODINE, 2).buildFor(recipeSaver, Minestuck.MOD_ID);
 		GristCostRecipeBuilder.of(MSItems.COMPUTER_PARTS.get()).grist(BUILD, 20).grist(RUST, 5).grist(GOLD, 1).build(recipeSaver);
 		
 		GristCostRecipeBuilder.of(MSItems.CRUXITE_APPLE.get()).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSItems.CRUXITE_POTION.get()).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.ANCIENT_THUMB_DRIVE.get()).build(recipeSaver);
```

Comment:
Move this over to where the rest of the gutter items are. Then give it a price proportional to the gutterball based on how much it improves the gutter size which is currently 10 times that of the gutterball. Will discuss that in an overview review comment